analytics-integrations/moengage/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply from: rootProject.file('gradle/integration.gradle')
+
+dependencies {
+
+    compile rootProject.ext.supportLib
+    compile 'com.google.android.gms:play-services-ads:7.3.0'
+    compile 'com.google.android.gms:play-services-gcm:7.3.0'
+    compile (name:'moengagesdk_v5.0.6', ext:'aar')
```

Comment:
As of now no. Its not on maven central or any other place. for now we have to do with a local aar file. I will move to maven dependency in future.